includes/class-wc-calypso-bridge-themes-setup.php

```diff
@@ -0,0 +1,278 @@
+<?php
```

Comment:
It looks like there is some formatting issues in this file that go against our PHP standards. Mostly indentation, but some lines with spacing issues like ` if (!$upload_file['error']) {`.